Readme.md

```diff
@@ -144,6 +144,18 @@ $ rm -rf $CHAIN/vendor/github.com/kr/pretty/.git
 It can change the behavior of cp and put the files
 in the wrong place.)
 
+Since this is a publicy-available open source project, there are some copyright
+issues to keep in mind:
+
+1. Don't vendor packages that are licensed under any flavor of the GPL.
```

Comment:
Like you suggested offline, maybe better to explicitly list the licenses we're concerned about (while still broadly including any other license with similar terms).

Like, "Don't vendor packages that are licensed under GPL, LGPL, AGPL, or any other license with similar terms. When in doubt, leave it out (or ask)." ?